coalib/output/Logging.py

```diff
@@ -60,14 +60,21 @@ def configure_logging(new_log_level=logging.INFO, incremental=False,
                 'stream': sys.stdout if stdout else sys.stderr,
                 'level': LOG_LEVEL.reverse.get(new_log_level)
             },
+            'file-handler': {
+                'class': 'logging.FileHandler',
+                'formatter': 'no-color',
+                'filename': '.coala.log',
+                'mode': 'w',
+                'level': 'DEBUG'
```

Comment:
yeah alright, I just thought we should allow it to be configurable anyway. Though lgtm first :+1: